globus_portal_framework/templates/components/footer.html

```diff
@@ -1,3 +1,11 @@
 <!-- Footer -->
+{% load static %}
 <footer>
+    <nav class="navbar navbar-expand-lg navbar-dark bg-dark navbar-base-nav">
+        <div style="text-align: center; padding: 10px; display: flex; justify-content: center;">
+            <a href="https://www.globus.org" target="_blank">
+                <img src="{% static 'globus-portal-framework/images/powered-by-GO-2012.png'%}" alt="Powered by Globus.org" style="height: 50px">
+            </a>
+        </div>
+    </nav>
```

Comment:
I think we could leave these changes off the older-style templates. That allows people to downgrade and keep the original look-and-feel of their site without also downgrading DGPF versions.

Reply:
I agree! Got them removed.